src/main.h

```diff
@@ -287,6 +306,38 @@ bool sendDiscoveryConnectivity()
     return false;
 }
 
+bool sendDiscoveryOccupancy()
+{
+    if (!discovery) return true;
+    if (pirsensor or radarsensor) 
+    {
+        String discoveryTopic = "homeassistant/binary_sensor/espresense_" + room + "/occupancy/config";
+
+        DynamicJsonDocument doc(1200);
+        char buffer[1200];
+
+        doc["~"] = roomsTopic;
+        doc["name"] = "ESPresense " + room + " Occupancy";
+        doc["unique_id"] = WiFi.macAddress() + "_occupancy";
+        doc["availability_topic"] = "~/status";
+        doc["stat_t"] = "~/occupancy";
```

Comment:
I think since we're using device_class motion and turning on an off quickly we should use ~/motion to be consistent.  Occupancy would work better if we truly knew if the room was occupied or not.